README.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```
+A clearfix is used to position an image or the like into a container when it spills out because it is too big.  If the image is floated then it will sort of jump out of the borders.  By adding a clearfix property and setting an overflow to auto it will correct the problem.
```

Comment:
What you have here is correct, but note the question also asked for an example of a clearfix!